src/test/java/com/datastax/cdm/feature/TrackRunTest.java

```diff
@@ -0,0 +1,18 @@
+package com.datastax.cdm.feature;
```

Comment:
I'm a bit surprised how the RAT checks didn't flag this as this is missing the copyright statement up at the top

Reply:
I resoled the issue, but good Q.